### PR TITLE
Create NuGet package for msm file

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**

### CODE REVIEW

File: src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.NuGet.csproj [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<Project>
+  <PropertyGroup>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <TargetExt>.nupkg</TargetExt>
+    <PackageId>Microsoft.InstrumentationEngine.Module</PackageId>
```
Comment (wiktork) [inline src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.NuGet.csproj]: consider MergeModule

File: src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.nuspec [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <owners>$authors$</owners>
+    <projectUrl>https://github.com/Microsoft/CLRInstrumentationEngine</projectUrl>
+    <license type="file">$licenseFile$</license>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Microsoft Instrumentation Engine Package containing the MergeModule</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+  </metadata>
+  <files>
+    <file src="$bindir$\InstrumentationEngine.Module_x64.$version$$msisuffix$.msm" target="Tools" />
```
Comment (wiktork) [inline src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.nuspec]: Is tools the right target here? Perhaps just content?

File: src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.NuGet.csproj [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <RootNamespace>Microsoft.InstrumentationEngine.Module</RootNamespace>
+    <NoWarn>2008</NoWarn>
+    <Platforms>x64</Platforms>
+  </PropertyGroup>
+  <PropertyGroup>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include="$(BinRoot)\$(Configuration)\x64\InstrumentationEngine.Module_x64.$(PackageVersion)$(MsiSuffix).msm">
+      <Link>InstrumentationEngine.Installer_x64.$(PackageVersion)$(MsiSuffix).msm</Link>
```
Comment (WilliamXieMSFT) [inline src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.NuGet.csproj]: >Installer_x64 [](start = 34, length = 13)

Module

File: src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.nuspec (hunk 2 above)
Comment (WilliamXieMSFT) [reply src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.nuspec]: From [https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#from-a-convention-based-working-directory](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#from-a-convention-based-working-directory)

It does seem like Content is the right folder for this.

---
In reply to: [354496267](https://github.com/microsoft/CLRInstrumentationEngine/pull/178#discussion_r354496267) [](ancestors = 354496267)

**Comment by WilliamXieMSFT** [comment]
>     <PackageId>Microsoft.InstrumentationEngine.Installer</PackageId>

Since we're not going to be publishing or using the nuget containing the installer, I'll remove this project & nuspec

---
Refers to: src/InstrumentationEngine.Installer.NuGet/Installer/InstrumentationEngine.Installer.NuGet.csproj:8 in da0b155. [](commit_id = da0b1550ebe6388d04df12a900f081d324d5e017, deletion_comment = False)

**Pull request merged**